11ty/definitions/r-word.md

```diff
@@ -0,0 +1,33 @@
+---
+title: R-word
+slug: r-word
+defined: true
+speech: noun
+flag:
+  level: avoid
+  text: ableist slur
+---
+
+Previously, the r-word was used in a medical context as an umbrella term to describe people with intellectual and developmental disabilities (I/DD).
+
+In modern use, the r-word is generally used as a denigrating term for a person with an I/DD, or to compare someone _without_ an I/DD to someone _with_ an I/DD in a derogatory sense.
+
+## Issues
+
+While the term was previously used in a medical context, use of the word has transitioned to having a negative connotation in modern language. Changes are being made in both the medical and legal fields to reflect that conversational shift.
+
+[Rosa's Law was passed in 2010](https://www.congress.gov/111/plaws/publ256/PLAW-111publ256.pdf) to eliminate all references to "mental retardation" in U.S. federal law, replacing them with "intellectual disability".
+
+The [DSM-5 update in 2013](https://www.psychiatry.org/File%20Library/Psychiatrists/Practice/DSM/APA_DSM-5-Intellectual-Disability.pdf) replaced "mental retardation" with "intellectual disability".
+
+## Impact
+
+Regardless of whether r-word is being used to describe someone with or without an I/DD, use of the word contributes to the dehumanization, stigmatization, and negative perceptions of people with an I/DD.
```

Comment:
The dictionary adheres to international English. Could you please change the z's in dehumanisation and stigmatisation to s's? Thank you!